vuepress/docs/v6.3/tutorials/devops/installation/open-shift/openshift-install-by-operator.md

```diff
@@ -0,0 +1,57 @@
+
+## Prerequisites
+- A 4.6 (or higher) OpenShift installation 
+  -  For simplicity the tutorial instructions are for a local Code Ready Containers (CRC) instance but this will require significant dedicated resources, e.g. at least 4 vCPU and 10GB RAM. In some environments you may need to start CRC using `crc start -n 8.8.8.8` in order to install operators. 
+- Cluster-admin access to OpenShift for initial installation of the Entando Operator
+- Familiarity with the OpenShift console and operation
+
+## Scenario 1
+The initial scenario is to deploy the operator and Entando application in a single namespace. We'll start with the smallest application footprint which uses an embedded database.
+1. Locate the `Entando Operator` in the `Operators → OperatorHub` using the Filter feature.
+2. Click `Install` to view the `Entando Operator` install options. 
+3. Select `A specific namespace on the cluster` for the `Installation mode`.
+4. Choose an empty namespace for `Installed Namespace`. You can create one from `Home → Projects` first, if needed, e.g. `entando`. 
+5. Click `Install` to install the operator into your target namespace.
+6. The install may take a few minutes to complete after which you can click `View Operator` to see the operator in your namespace, or you can go to `Operators → Installed Operators` at any point and select it from there. 
+7. Now go to `Entando Composite Application` and click `Create instance`
+8. Change the `Dbms Override` to `embedded`. This is the lightest and quickest way to test a full Entando application but you'll need to modify the yaml next.
+9. (Optional) If you're working in anything but a single project environment, you'll need to provide an `Ingress Host Name Override` specific to your namespace, e.g. `entando-project-1.<my-base-openshift-url>.` In CRC you can keep the default `entando.apps-crc.testing` for your first project.
+10. Switch to the YAML view and remove the following section since it isn't needed with an embedded database.
+```
+    - kind: EntandoDatabaseService
+      metadata:
+        name: inline-entando-database-service
+      spec:
+        createDeployment: true
+``` 
+11. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources. 
+12. Go to `Entando Composite Application → my-entando-composite-app` to check the status of the deploy or its `Events` tab to follow its progress.
+
+See the [Next Steps](#next-steps) below to continue your work with Entando or move on to the next Scenario if you want to try another variation.
+ 
+ ## Scenario 2 - Shared PostgreSQL database
```

Comment:
Leading space here i think will break the header formatting

Reply:
Got it, thanks. Must have fixed it locally since it's gone in my copy.